en/docs/services/create-your-first-service.md

```diff
@@ -1,177 +1,250 @@
+
+6. In the **Operation** field, select **Get Country Population**. Then enter the following information in the other fields that appear.
+
+    1. In the **Country Code** field, enter `country`.
     
-4. To store the response from the public API to your service, follow this procedure:
+    2. In the **Date** field, enter `date`.
+    
+    3. Click **Optional**, and in the **Format** field, enter `json`.
+
+    4. In the **Response Variable Name** field, enter `populationByCountry`
 
-    1. Click the last **+** icon in your low-code diagram.
+    Save the information.
 
-        ![Add New Statement](../assets/img/services/add-custom-statement.png){.cInlineImage-half}
+7. Now let’s extract the population from the response and store it in a variable.
+
+    To do this, click the last **+** icon in the low-code diagram and click **Variable**.
     
-    2. Click **Other** and then enter the following in the **Statement** field:
+    Enter information as follows:
     
-        ```ballerina
-        json[] jsonArray = <json[]>jsonPayload;
-        json[] response = [];
-        ```
-        
-    3. Click **Save**.
+    | **Field**      | **Value**                      |
+    |----------------|--------------------------------|
+    | **Type**       | `int`                          |
+    | **Name**       | `population`                   |
+    | **Expression** | `(populationByCountry is worldbank:CountryPopulationArr ? populationByCountry[0]?.value ?: 0 : 0) / 1000000` |
     
-5. To filter the dates on which the active cases have exceeded 5,000, follow this procedure: 
-
-    1. Click the last **+** icon in your low-code diagram.
+    Save the information.
     
-    2. Click **ForEach**.
+## Step 4: Calculate the summary of the COVID-19 cases
 
-    3. In the **Iterable Expression** field, enter `jsonArray`and then click **Save**.
-    
-    4. Click the **+** icon just below the last ForEach statement you added.
+Now let’s calculate the total cases per million in the population based on the COVID-19 statistics and the population data that you have collected. 
+
+To do this, click the last **+** icon in the low-code diagram and click **Variable**.
+
+Enter information as follows:
+
+| **Field**      | **Value**                  |
+|----------------|----------------------------|
+| **Type**       | `var`                      |
+| **Name**       | `totalCasesPerMillion`     |
+| **Expression** | `totalCases / population`  |
+
+Save the information.
+
+## Step 5: Build the JSON payload and respond
+
+To build the JSON payload and respond, follow these steps:
+
+1. To build the `json` payload with data of the total cases per million in the population, add a variable.
 
-        ![Add Custom Statement After Foreach Statement](../assets/img/services/add-custom-statement-after-foreach-statement.png){.cInlineImage-half}
+    Click the last **+** icon in the low-code diagram and click **Variable**. Then enter information as follows:
```

Comment:
"In the **Statements** tab" part is missing

Reply:
Did not mention it because the **Statements** tab is open by defaults and this statement type is visible as soon as the dialog box opens.